app/locales/en.json

```diff
@@ -119,7 +119,7 @@
   "Everyone who has submitted a report, also has:": "Everyone who has submitted a report, also has:",
   "Total number of reports": "Total number of reports",
   "Positive test results and close contact": "Positive test results and close contact",
-  "Clarifications data basis": "Clarifications data basis",
+  "Clarifications data basis": "Förtydliganden angående databasen",
```

Comment:
Is this translation intended to be in this file?

Reply:
I wouldn't think so either, since it's `en.json`.

Reply:
Oops, sorry. Was to tired and added it to the wrong file. It's now in the `se.json`.